colors/termschool.vim

```diff
@@ -127,3 +127,5 @@ hi SpellRare ctermfg=0 ctermbg=225 guifg=#000000 guibg=#ffdfff
 hi TabLineFill ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
 hi TabLineSel ctermfg=NONE ctermbg=59 cterm=NONE guifg=NONE guibg=#3f4b52 gui=NONE
 hi TabLine ctermfg=NONE ctermbg=233 cterm=NONE guifg=NONE guibg=#121212 gui=NONE
+hi SignColumn     ctermbg=NONE guibg=#2e373b
```

Comment:
I couldn't find this guibg in https://upload.wikimedia.org/wikipedia/en/1/15/Xterm_256color_chart.svg. Please choose a close that is close to this one and match ctermbg to it.